src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--1 calendar__day--mon--1"></div>
```

Comment:
```suggestion
      <div class="calendar__day></div>
```
There should be only 1 class use nth-child